### PR TITLE
github: add workflow for building olsrd

### PR DESCRIPTION
**Pull request by PolynomialDivision**

### CODE REVIEW

**Comment by booo** [comment]
Could you provide us with some information about the intentions of this pull request in the commit message and the pull request?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: olsrd
+      - name: Install dependencies
+        run: sudo apt-get install build-essential dpkg-dev bison flex
+      - name: Compile olsrd
+        run: cd olsrd && make
```
Comment (booo) [inline .github/workflows/build.yml:17]: This should be "cd olsrd && make build_all". Or we should have two build steps. One for olsrd and one for the plugins.

Comment (PolynomialDivision) [reply .github/workflows/build.yml:17]: I will fix that.

**Comment by booo** [comment]
Thanks for your contribution. Would you mind taking a look at building openwrt on different targets such as windows or such?

**Pull request merged**